### ISSUE TITLE
Windows: Filename too long

### ISSUE THREAD

**Issue by monsdar** [lasote/conan-boost#17]
I'm getting a `filename too long` warn when unzipping `Boost/1.59.0@lasote/stable` on my Win7 machine. It's just a doc file, but I thought reporting it could help none the less.

```
Error extract boost_1_59_0/libs/geometry/doc/html/geometry/reference/spatial_indexes/boost__geometry__index__rtree/rtree_parameters_type_const____indexable_getter_const____value_equal_const____allocator_type_const___.html
```

The build itself was successful.

**Comment by ksmith-mm** [comment]
I am seeing this as well but more than initially reported. My conan install path is only set to E:\conan\. Attaching a txt file with a list of them.
[conan-boost-long-filenames.txt](https://github.com/lasote/conan-boost/files/625604/conan-boost-long-filenames.txt)

**Comment by lasote** [comment]
@ksmith-mm It was fixed in Boost 1.60.0 package (with short_paths = True attribute). Do you need to use 1.58.0? I could make the patch to this version.
